libunwind/src/libunwind.cpp

```diff
@@ -288,6 +288,11 @@ void __unw_remove_dynamic_fde(unw_word_t fde) {
 
 _LIBUNWIND_HIDDEN
 bool logAPIs() {
+#ifdef RUST_SGX
```

Comment:
Looks like these APIs are used only in debug builds, can these changes be avoided?

Reply:
removed these changs, we never define the NDEBUG flag so they are never compiled in our use case.